docs/content/config/advanced/ipv6.md

```diff
@@ -2,47 +2,14 @@
 title: 'Advanced | IPv6'
 ---
 
-## Background
+## Not Available
+docker-mailserver doesn't currently support iPv6 networking! Trying to get mails using iPv6 will result with postfix not being able to find your hostname.
```

Comment:
I don't think this is 100% correct. It is probably not even a DMS issue, but a Docker issue. I could try setting up a Kubernetes dual-stack cluster which supports IPv6 just fine and DMS should work in this setup too. @casperklein @polarathene do you think this really is accurate?

Reply:
> It is probably not even a DMS issue, but a Docker issue.
> @polarathene do you think this really is accurate?

The previous content was more accurate about the "why" and should remain really (_the gateway IP varies IIRC, dependent on what the docker network got assigned_).

The #2927 referenced issue is the one I was intending to respond to, but got side-tracked with all the test refactoring in jan :sweat_smile: I recently [shared details on how to get IPv6 working properly](https://github.com/docker-mailserver/docker-mailserver/pull/3057#issuecomment-1416706615), although I haven't applied it for DMS myself yet to verify that resolves the issue.

Probably better to update the docs with information from there as the IPv6 NAT support wasn't available when these docs were written IIRC.

Reply:
> do you think this really is accurate?

Afaik, none of us is using IPv6. So we can't properly test/reproduce any errors that are reported. Given the amount of IPv6 related issue, I am not sure if we should advertise it to be functional. So a warning seems reasonable to me. The wording however is a bit harsh and might be adjusted.